app/controllers/sections/info/DestinationTypeController.scala

```diff
@@ -52,33 +52,32 @@ class DestinationTypeController @Inject()(
                                            val userAllowList: UserAllowListAction
                                          ) extends BasePreDraftNavigationController with AuthActionHelper with PreDraftAuthActionHelper {
 
-  def onPreDraftPageLoad(ern: String, arc: String, mode: Mode): Action[AnyContent] =
+  def onPreDraftPageLoad(ern: String, arc: String): Action[AnyContent] =
     authorisedWithPreDraftDataUpToDateMovementAsync(ern, arc) { implicit request =>
-      renderView(Ok, fillForm(DestinationTypePage, formProvider()), mode)
+      renderView(Ok, fillForm(DestinationTypePage, formProvider()))
     }
 
-  def onPreDraftSubmit(ern: String, arc: String, mode: Mode): Action[AnyContent] =
+  def onPreDraftSubmit(ern: String, arc: String): Action[AnyContent] =
     authorisedWithPreDraftDataUpToDateMovementAsync(ern, arc) { implicit request =>
       formProvider().bindFromRequest().fold(
         formWithErrors =>
-          renderView(BadRequest, formWithErrors, mode),
+          renderView(BadRequest, formWithErrors),
         value =>
-          savePreDraftAndRedirect(DestinationTypePage, value, mode)
+          createDraftEntryAndRedirect(DestinationTypePage, value)
       )
     }
 
-  private[info] def renderView(status: Status, form: Form[_], mode: Mode)(implicit request: DataRequest[_]): Future[Result] = {
+  private[info] def renderView(status: Status, form: Form[_])(implicit request: DataRequest[_]): Future[Result] = {
     Future.successful(
       request.dispatchPlace match {
         case _ if request.userTypeFromErn != NorthernIrelandWarehouseKeeper =>
           // GB ERN or XIRC ERN
-          status(view(GreatBritain, form, controllers.sections.info.routes.DestinationTypeController.onPreDraftSubmit(request.ern, request.arc, mode)))
+          status(view(GreatBritain, form, controllers.sections.info.routes.DestinationTypeController.onPreDraftSubmit(request.ern, request.arc)))
         case Some(dispatchPlace) =>
-          // XIWK ERN, dispatchPlace is known
-          status(view(dispatchPlace, form, controllers.sections.info.routes.DestinationTypeController.onPreDraftSubmit(request.ern, request.arc, mode)))
+          status(view(dispatchPlace, form, controllers.sections.info.routes.DestinationTypeController.onPreDraftSubmit(request.ern, request.arc)))
         case None =>
-          // XI ERN, dispatchPlace is unknown
-          Redirect(controllers.sections.info.routes.DispatchPlaceController.onPreDraftPageLoad(request.ern, request.arc, mode))
+          //  dispatchPlace is unknown
+          Redirect(controllers.sections.info.routes.DispatchPlaceController.onPreDraftPageLoad(request.ern, request.arc, NormalMode))
```

Comment:
DispatchPlacePage currently has a TODO which means that when the user is a XIWK then it will always fall in the None block. The DispatchPlace page routing isn't in the copydeck